recipe/meta.yaml

```diff
@@ -22,7 +22,7 @@ requirements:
   run:
     - python >=3.8,<3.11
     - pip
-    - coiled >=0.2.54
+    - coiled >=0.2.58
```

Comment:
Looking at the CI failures in https://github.com/coiled/coiled-runtime/actions/runs/3831858841/jobs/6521497028 and other jobs in the same run, should we update `coiled` to `0.2.58`?

Reply:
0.2.58 is already installed by `>=0.2.54` , this is a race condition I have a fix for